Tools/CMakeLists.txt

```diff
@@ -39,6 +39,11 @@ configure_file( "${CMAKE_SOURCE_DIR}/cmake/FindJsonGenerator.cmake.in"
                 @ONLY
 )
 
+configure_file( "${CMAKE_SOURCE_DIR}/cmake/FindDocGenerator.cmake.in"
```

Comment:
Not sure we need to introduce another file, since the DocGenerator is JsonGenerator.